ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/errors/PaywallValidationError.kt

```diff
@@ -93,14 +92,6 @@ internal sealed class PaywallValidationError : Throwable() {
             PaywallValidationErrorStrings.MISSING_PACKAGE
                 .format(missingPackageId, offeringId, allPackageIds.joinToString())
     }
-    data class MissingAllPackages(
```

Comment:
Might be good having a paywall without any packages so we can confirm it works as expected with no packages but from looking at the code and from what I remember, I don't think there should be any issues later 👍

Reply:
I did test this locally and we will have a new template that will have this that will get added to our cross platform validation soon 💪